bundles/org.openhab.binding.x/src/main/java/org/openhab/binding/x/internal/XHandler.java

```diff
@@ -117,102 +115,102 @@ private void refresh() {
             }
             Twitter localClient = client;
             if (localClient != null) {
-                ResponseList<Status> statuses = localClient.getUserTimeline();
-                if (!statuses.isEmpty()) {
-                    updateState(CHANNEL_LASTTWEET, StringType.valueOf(statuses.get(0).getText()));
+                ResponseList<Status> statuses = localClient.v1().timelines().getUserTimeline();
+                if (statuses.size() > 0) {
```

Comment:
This also undos #15576:

```suggestion
                if (!statuses.isEmpty()) {
```